### ISSUE TITLE
[d-message-strip][Core]   Screen Reader ACC-264.1(Level A) Message strip information is not read

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13229]
Is it a bug ?
Yes 

version v0.54.2

https://stackblitz.com/run?file=src%2Fapp%2Fmessage-strip-example.component.html

Screen reader cannot read the message tip(error ,warning, success..)

![Image](https://github.com/user-attachments/assets/51589ccf-1ae9-4269-8f5e-ed8af950add5)

**Comment by droshev** [comment]
@Meggie886 Can you post what JAWS announcement was and what expectation is? BTW the stackblitz link is not correct.

**Comment by Meggie886** [comment]
ACC-264.1: Are all UI elements presented correctly using assistive technologies?
Follow the test case.
Identify all [elements on the screen](https://wiki.one.int.sap/wiki/display/Acc/ACC-264+Details#ACC264Details-identifyelements) that are interactive or/and have a semantic meaning or serve only decoration purposes.
Check if [element role, name, state(s), values and other element attributes](https://wiki.one.int.sap/wiki/display/Acc/ACC-264+Details#ACC264Details-acc2641howto) for interactive and semantic/non-semantic elements can be determined by [assistive technology (screen reader)](https://wiki.one.int.sap/wiki/display/Acc/ACC-264+Details#ACC264Details-usingscreenreader) when they have keyboard or virtual focus on desktop devices or been navigated using explore by touch navigation on mobile devices.
Realized
(1) All UI elements are presented correctly using assistive technologies
Partly realized
(1) Some UI elements are not presented correctly using assistive technologies: <list of UI elements>
Not realized
(1) None of the UI elements is presented correctly using assistive technologies

Expectation : Screen reader can read the type pf  Message strip


And the link I don't know why you cannot open, if you cannot just check Message Strip Component, any example is ok

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us what is exactly expected that we should update? thank you very much.

**Comment by nikolay-kolarov** [comment]
Hi @droshev,
The icons are designed to be decorative, so you need to add the semantic of the Info bar, so that it can be reached with assistive technologies (in reading mode or with focus).
You can have a look at the message strip component: https://sap.github.io/ui5-webcomponents/components/MessageStrip/


For the reading mode:

![Image](https://github.com/user-attachments/assets/f4f7cf09-48ae-4d80-85c0-9e872800f634)

For the button on focus:

![Image](https://github.com/user-attachments/assets/d899c56f-a547-4058-8184-9520584bc62a)

The text "information bar" is by spec.

Best Regards,
Nikolay
